js/src/logger.ts

```diff
@@ -827,7 +827,7 @@ export function updateSpan({
 }
 
 interface ParentSpanIds {
-  spanId: string;
+  spanId?: string;
```

Comment:
Discussed offline that we can undo these changes since they are not (yet) necessary.